feature-wallet-connect-impl/src/main/java/io/novafoundation/nova/feature_wallet_connect_impl/domain/session/RealWalletConnectSessionInteractor.kt

```diff
@@ -136,6 +136,15 @@ class RealWalletConnectSessionInteractor(
         return walletConnectSessionRepository.getSessionAccount(sessionTopic)
     }
 
+    override fun activeSessionsFlow(metaId: Long): Flow<List<WalletConnectSession>> {
+        return walletConnectSessionRepository.sessionAccountsByMetaIdFlow(metaId).map { sessionAccounts ->
+            val activeSessions = Web3Wallet.getListOfActiveSessions()
+            val metaAccount = accountRepository.getMetaAccount(metaId)
```

Comment:
Its better to subcribe to `metaAccount` separately and then combine it with `walletConnectSessionRepository.sessionAccountsByMetaIdFlow` to always have the latest `metaAccount` state